tests/Core/Components/Switch/FluentSwitchTests.razor

```diff
@@ -0,0 +1,103 @@
+﻿@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Tests.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Utilities
+@using System.ComponentModel.DataAnnotations
+@using Xunit;
+@inherits TestContext
+
+@code
+{
+    public FluentSwitchTests()
```

Comment:
The Obsolete attributes (ChildContent, CheckedMessage and UncheckedMessage) must be unit tested.

You can run the file `_StartCodeCoverage.cmd` to validate if all code lines a covered.

Reply:
Ok, Tests added for ChildContent, CheckedMessage and UncheckedMessage.

Reply:
More tests added and `_StartCodeCoverage.cmd` run. 100% on branch coverage and a bit like `FluentCheckBox` on Line coverage.